components/generic_modal.scss

```diff
@@ -118,3 +114,98 @@
         }
     }
 }
+
+.StatusModal {
```

Comment:
Can we put this css aside the new component as `components/custom_status_input_modal/custom_status_input_modal.scss` and import it directly from the modal typescript file?